$NetBSD: patch-configure,v 1.4 2024/02/21 10:21:01 wiz Exp $

- Do not store configure options as these might be polluted with workdir.

--- configure.orig	2024-02-21 09:11:27.578146251 +0000
+++ configure
@@ -3752,7 +3752,7 @@ ac_config_headers="$ac_config_headers in
 # save the configure arguments
 #
 
-printf "%s\n" "#define NETSNMP_CONFIGURE_OPTIONS \"${NETSNMP_CONFIGURE_OPTIONS-$ac_configure_args}\"" >>confdefs.h
+printf "%s\n" "#define NETSNMP_CONFIGURE_OPTIONS \"\"" >>confdefs.h
 
 CONFIGURE_OPTIONS="\"$ac_configure_args\""
 
